code/game/gamemodes/cult/runes.dm

```diff
@@ -952,71 +780,164 @@ var/list/teleport_runes = list()
+		var/obj/machinery/shield/N = new(get_turf(src))
+		N.name = "Invoker's Shield"
+		N.desc = "A weak shield summoned by cultists to protect them while they carry out delicate rituals"
+		N.color = "red"
+		N.health = 20
+		N.mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+		new_human.key = ghost_to_spawn.key
+		SSticker.mode.add_cultist(new_human.mind, 0)
+		to_chat(new_human, "<span class='cultitalic'><b>You are a servant of the [SSticker.cultdat.entity_title3]. You have been made semi-corporeal by the cult of [SSticker.cultdat.entity_name], and you are to serve them at all costs.</b></span>")
+
+		while(!QDELETED(src) && !QDELETED(user) && !QDELETED(new_human) && (user in T))
+			if(new_human.InCritical())
+				to_chat(user, "<span class='cultitalic'>You feel the connection to [new_human] severs as they are destroyed.</span>")
+				break
+			if(user.stat || affecting.health <= 40)
+				to_chat(user, "<span class='cultitalic'>Your body can no longer sustain the connection, and your link to the spirit realm fades.</span>")
+				break
+			user.apply_damage(0.1, BRUTE)
+			sleep(1)
+		qdel(N)
+		ghosts--
+		if(new_human)
+			new_human.visible_message("<span class='warning'>[new_human] suddenly dissolves into bones and ashes.</span>", \
+									  "<span class='cultlarge'>Your link to the world fades. Your form breaks apart.</span>")
+			for(var/obj/I in new_human)
+				if(I.flags & NODROP)
+					qdel(I)
+				else
+					I.forceMove(new_human.drop_location())
+			new_human.dust()
+	else if(choice == "Ascend as a Dark Spirit")
+		affecting = user
+		affecting.add_atom_colour(RUNE_COLOR_DARKRED, ADMIN_COLOUR_PRIORITY)
+		affecting.visible_message("<span class='warning'>[affecting] freezes statue-still, glowing an unearthly red.</span>", \
+						 "<span class='cult'>You see what lies beyond. All is revealed. In this form you find that your voice booms louder and you can mark targets for the entire cult</span>")
+		var/mob/dead/observer/G = affecting.ghostize(1)
+		var/datum/action/innate/cult/comm/spirit/CM = new
+		//var/datum/action/innate/cult/ghostmark/GM = new
+		G.name = "Dark Spirit of [G.name]"
+		G.color = "red"
+		CM.Grant(G)
+		//GM.Grant(G)
+		while(!QDELETED(affecting))
+			if(!(affecting in T))
+				user.visible_message("<span class='warning'>A spectral tendril wraps around [affecting] and pulls [affecting.p_them()] back to the rune!</span>")
+				Beam(affecting, icon_state="drainbeam", time=2)
+				affecting.forceMove(get_turf(src)) //NO ESCAPE :^)
+			if(affecting.key)
+				affecting.visible_message("<span class='warning'>[affecting] slowly relaxes, the glow around [affecting.p_them()] dimming.</span>", \
+									 "<span class='danger'>You are re-united with your physical form. [src] releases its hold over you.</span>")
+				affecting.Weaken(3)
+				break
+			if(affecting.health <= 10)
+				to_chat(G, "<span class='cultitalic'>Your body can no longer sustain the connection!</span>")
```

Comment:
You've been using 'cult italic' thus far. Now you're using 'cultitalic'. Current code uses 'cultitalic' and one case of 'cult italic'.

Reply:
fixed in next commit, cultitalic is the intended span